idb/postgres/postgres_migrations.go

```diff
@@ -96,8 +96,7 @@ func (db *PostgresIndexerDb) runAvailableMigrations(migrationStateJson string) (
 		return err
 	}
 
-	// TODO: Start it in the background instead of blocking
-	db.migration.Start()
+	go db.migration.Start()
```

Comment:
The caller shouldn't be aware of the internals of `Migration`. Consider moving the content of `Start()` to an internal function to `Migrations` and run this it as a goroutine from `Start()`. As an alternative, if you want the caller to decide whether to run it as blocking or not, consider changing the name of `Start` to `RunMigrations` (see the comment in the other PR).

Reply:
Thanks, I'll rename the function. I either need a "done" channel or to have this function block.